.github/workflows/cli-tests.yaml

```diff
@@ -94,6 +94,8 @@ jobs:
             os: "${{ inputs.run-mac-tests && 'dummy' || 'macos-latest' }}"
           - is-main: true
             run-example-tests: false
+          - run-example-tests: true
+            os: macos-latest
```

Comment:
can we make this line: `os: "${{ inputs.run-mac-tests && 'dummy' || 'macos-latest' }}"` ?

that way, we can kick off manual workflows that do run macos tests